README.md

```diff
@@ -1,45 +1,36 @@
+[SensioLabs Security Check Web service](http://security.sensiolabs.org/) and the [Security Advisories Database](https://github.com/FriendsOfPHP/security-advisories).
 
-### Requirements
+BSD 3-clause [License](https://github.com/bringyourownideas/silverstripe-composer-security-checker/blob/master/license.md)
 
-* SilverStripe Framework ^3.0
-* SilverStripe QueuedJobs *
+## Requirements
 
+* SilverStripe Framework ^4
+* SilverStripe QueuedJobs ^4
 
-### Installation
+### Suggested Module
 
-The following installation commands includes schedulding a queuedjob to populate the data. Run the following command to install this package as a development dependency:
+This module will automatically amend the SiteSummary report provided by the [SilverStripe Maintenance module](https://github.com/bringyourownideas/silverstripe-maintenance), adding alerts if security updates are present for installed modules.
 
-```
-composer require bringyourownideas/silverstripe-composer-security-checker dev-master --dev
-php ./framework/cli-script.php dev/build
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
-```
+## Installation
 
-*or* as general dependency:
+The following installation commands includes schedulding a queuedjob to populate the data. Run the following command to install this package as a development dependency:
 
 ```
 composer require bringyourownideas/silverstripe-composer-security-checker dev-master
-php ./framework/cli-script.php dev/build
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
+vendor/bin/sake dev/build
+vendor/bin/sake dev/tasks/ProcessJobQueueTask
 ```
 
-**Recommendation is to only install this as dev dependency!**
-
-
-### Usage
-
-The information gets automatically updated via a queuedjob on dev/build. You will need to run the queuedjobs task to get the information.
-
-Use the information in your database (Table "ComposerSecurityVulnerability") as you like. Please be careful how you expose this information.
-
+## Usage
 
-## MISC: [Future ideas/development, issues](https://github.com/bringyourownideas/silverstripe-composer-security-checker/issues), [Contributing](https://github.com/bringyourownideas/silverstripe-composer-security-checker/blob/master/CONTRIBUTING.md), [License](https://github.com/bringyourownideas/silverstripe-composer-security-checker/blob/master/license.md)
+The information gets updated via a BuildTask, which in turn can be run via a queuedjob.
+You will need to set up a scheduled process (e.g. `cron`) to run either the buildtask directly, or the task to process the queuedjobs queue in order to refresh the information.
 
-[1]: http://security.sensiolabs.org/
-[2]: https://github.com/FriendsOfPHP/security-advisories
+Use the information is stored in the `SecurityAlert` object, and can be consumed as needed. Please be careful how you expose this information. If the SilverStripe Maintenance module is present, a relationship will be connected between `Package` and `SecurityAlert`.
```

Comment:
It's a shame really that this wasn't done in a separate task against the 1 branch. We'll need to backport these readme updates at some stage into the 1 branch.